### PR TITLE
add(slb_master_slave_server_group): update slb sdk and add slb master slave server group resource & data source

### PR DESCRIPTION
**Pull request by FireWoring**
resource and data source are tested as follow:

屏幕快照 2019-08-07 下午12 25 47

### CODE REVIEW

File: website/docs/r/slb_master_slave_server_group.html.markdown [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+-> **NOTE:** One master slave server group can be attached with tcp/udp listeners in one load balancer.
+
+-> **NOTE:** One Classic and Internet load balancer, its master slave server group can add Classic and VPC ECS instances.
+
+-> **NOTE:** One Classic and Intranet load balancer, its master slave server group can only add Classic ECS instances.
+
+-> **NOTE:** One VPC load balancer, its master slave server group can only add the same VPC ECS instances.
+
+-> **NOTE:** Available in 1.53.0+
```
Comment (YY-tomorrow) [inline website/docs/r/slb_master_slave_server_group.html.markdown]: still

File: website/docs/r/slb_master_slave_server_group.html.markdown [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+* `is_backup` - (Optional)  Determine if the server is executing. Valid value 0, 1. 
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the master slave server group.
+* `load_balancer_id` - The Load Balancer ID which is used to launch a new master slave server group.
+* `name` - The name of the master slave server group.
+* `servers` - Two ECS instances that have be added.
```
Comment (YY-tomorrow) [inline website/docs/r/slb_master_slave_server_group.html.markdown]: Only need id

File: alicloud/resource_alicloud_slb_master_slave_server_group.go [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"servers": {
+				Type:     schema.TypeSet,
+				Required: true,
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_slb_master_slave_server_group.go]: I think it should be Optional.

File: alicloud/service_alicloud_slb.go [hunk 4]
```diff
@@ -328,6 +348,30 @@ func (s *SlbService) WaitForSlbServerGroup(id string, status Status, timeout int
...
+		if err != nil {
+			if NotFoundError(err) {
+				if status == Deleted {
+					return nil
+				}
+			} else {
+				return WrapError(err)
+			}
+		}
+		if object.MasterSlaveServerGroupId == id {
```
Comment (xiaozhu36) [inline alicloud/service_alicloud_slb.go]: `if object.MasterSlaveServerGroupId == id && status != Deleted`

File: website/docs/d/slb_master_slave_server_groups.html.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_slb_master_slave_server_groups"
+sidebar_current: "docs-alicloud-datasource-slb-master-slave-server-groups"
+description: |-
+    Provides a list of master slave server groups related to a server load balancer to the user.
+---
+
+# alicloud\_slb\_master\_slave\_server\_group
```
Comment (xiaozhu36) [inline website/docs/d/slb_master_slave_server_groups.html.markdown]: alicloud\_slb\_master\_slave\_server\_groups?

**Pull request merged**